week-2/J-array-get-set/exercises2.js

```diff
@@ -7,6 +7,9 @@
 */
 
 var numbers = [2, 2, 3]; // Don't change this array literal declaration
+numbers.push(4);
+numbers.shift();
+numbers.unshift(1);
```

Comment:
This is correct, well done for finding an alternative solution. However can actually do this in 1 statement, using array indexes (e.g. `[0]`). Do you know how that might work?